compose-files/.env.example

```diff
@@ -0,0 +1,2 @@
+VERSION=v14
+SITENAME="site1.local"
```

Comment:
remove `.env.example` file add / use / change variables from

https://github.com/frappe/frappe_docker/blob/main/example.env

You can add SITENAME in example.env `FRAPPE_VERSION` and `ERPNEXT_VERSION` already exists there. You can use one from it instead of adding 1 more `VERSION`. Also example.env is updated on every release and the *_VERSION(s) are bumped as per the upstream release. You don't need to maintain that env further.